community/vendor-technical-contacts/teleport.md

```diff
@@ -0,0 +1,26 @@
+# Teleport
+
+## Documentation
+
+Documentation for Teleport Workload Identity can be found at https://goteleport.com/docs/enroll-resources/machine-id/workload-identity/
+
+## Support
+
+Enterprise customers should seek support via https://goteleport.com/support/
+
+OSS users can should seek support via the community Slack: https://goteleport.com/community-slack/
+
+## Contacts
+
+### Noah Stride (Technical && Community)
```

Comment:
Typo: Double ampersand (unless you want a logical AND) :)

Reply:
It's possible I'd spent a few too many hours coding before writing this 😭

Reply:
[57431e3](https://github.com/spiffe/spiffe/pull/298/commits/57431e33f63a2d2f736fedfcdd8ba884876bf8ef)